### ISSUE TITLE
Display nicknames rather than server names

### ISSUE THREAD

**Issue by stevesbrain** [sm00th/bitlbee-discord#55]
Not sure if this is closely related enough to #40 to warrant it being part of the same issue, but, channels seem to display the account name/username (not sure of the terminology) rather than the per-server nickname. Would be handy to have the per-server nick showing up instead :)

**Comment by sm00th** [comment]
This happens so because we have a single irc "server" those users "join", and they all show up in &bitlbee control channel.

If you join 2 servers with the same person on them but with 2 different per-server nicks, how do you propose we handle this situation in bitlbee? If we choose a per-server nickname at random and use that for both instances of that user, that would be confusing. If we use global nickname while all non-duplicate users have their per-server nicknames that would be confusing as well. The only solution that won't confuse bitlbee-discord users that I see is adding both nicnames to the control channel, which still might be a bit confusing, is probably a nightmare to implement and is generally a wrong approach in my opinion.

**Comment by stevesbrain** [comment]
No, you are correct of course - I just didn't think the problem through in relation to the bigger bitlbee picture! Thanks for explaining :)

**Comment by witheld9** [comment]
Suggesting re-open this- on certain servers I use, nicknames are contextual, informative, and changed frequently. On others they're used for comedic effect which is still useful. I'm no sure what the best solution is- but one that lets me see people's nicks as they intend for me to see them is necessary to me

**Comment by witheld9** [comment]
My suggestion would be to make this a channel setting. If enabled, messages are sent to that channel with the fake names, without actually creating any users with those name. The one cost of this is you can't query people in channels with that setting enabled but it seems like the cleanest approach

**Comment by fastfinge** [comment]
Could bitlbee-discord put the nickname as the real name on IRC? That way it would show up on /whois user. Also, scripts that wanted to parse it to make tooltips or change it on a per channel basis or whatever could easily do so client side.  And I feel like client side might be the best place for this to be handled.

**Comment by sm00th** [comment]
Reopening this due to relatively high demand, and this kinda makes more sense now that friendship_mode is there.

**Comment by rosborne018** [comment]
you could have the option to display the nickname after the username
like name:nick

**Comment by Alcaro** [comment]
What if you share two guilds? They're the same person, do they get two nicks or do they talk from wrong nick on the other guild?

What if said person then sends a PM? Which of them does it come from?

I'm sure it's solvable somehow, but like many other issues, it's not as simple as it seems.

**Comment by jlu5** [comment]
If we use the name:nick format I wouldn't mind having multiple clients created for one person (having them all forward to the same target).
